app/containers/CreateWallet/CreateWallet.jsx

```diff
@@ -107,14 +136,31 @@ export default class CreateWallet extends React.Component<Props, State> {
               onSubmit={this.createWalletAccount}
             >
               {option === 'IMPORT' && (
-                <PasswordInput
-                  value={wif}
-                  label="Private Key"
-                  onChange={e => this.setState({ wif: e.target.value })}
-                  placeholder="Private Key"
-                  autoFocus
-                />
+                <div>
+                  <div className={styles.SelectContainer}>
+                    <StyledReactSelect
```

Comment:
Sorry to be so nitpicky but I think for consistency we should actually use our "tabs" component here instead of the select drop down... You can see an example of this on the Login page container. I think this will look better here

Reply:
I like much better with tabs component. This is really a good improvement on the look and feel. Thanks for the suggestion!

Reply:
![image](https://user-images.githubusercontent.com/30392990/59983742-98050b80-9623-11e9-83e6-5e0ab0936243.png)

Reply:
@dauTT this is looking really good! Would we be able to use all upper case text in tabs (for consistency) and remove the "instructions" like in this mock here (you can leave out the last tab in this PR):
<img width="1222" alt="Screen Shot 2019-06-24 at 7 21 00 PM" src="https://user-images.githubusercontent.com/13072035/60135782-782f2e00-9770-11e9-9b0e-ff41dcf6cb09.png">